### PR TITLE
feat: accessibility updates from alpha

### PR DESCRIPTION
**Pull request by kabaros**
This is a cherry-pick for features already in `alpha` branch. Fixes:

- Button accessibility: https://dhis2.atlassian.net/browse/LIBS-565 
- Loaders accessiblity: https://dhis2.atlassian.net/browse/LIBS-560
- Tooltip accessibility: https://dhis2.atlassian.net/browse/LIBS-558
- Split button accessibility: https://dhis2.atlassian.net/browse/LIBS-566

### CODE REVIEW

File: components/tooltip/src/tooltip.js [hunk 1]
```diff
@@ -66,33 +64,63 @@ const Tooltip = ({
...
                     ref: popperReference,
                 })
             ) : (
                 <span
                     onMouseOver={onMouseOver}
                     onMouseOut={onMouseOut}
+                    onFocus={onFocus}
+                    onBlur={onBlur}
                     ref={popperReference}
+                    aria-describedby={open ? 'tooltipContenDhis2Ui' : ''}
```
Comment (alaa-yahia) [inline components/tooltip/src/tooltip.js:118]: Could you add `tabIndex={0}` to this span?
I forget to add it in my pr

File: components/tooltip/src/tooltip.js [hunk 2]
```diff
@@ -66,33 +64,63 @@ const Tooltip = ({
...
             {typeof children === 'function' ? (
                 children({
                     onMouseOver: onMouseOver,
                     onMouseOut: onMouseOut,
+                    onFocus: { onFocus },
+                    onBlur: { onBlur },
                     ref: popperReference,
                 })
             ) : (
                 <span
```
Comment (alaa-yahia) [inline components/tooltip/src/tooltip.js:112]: Could you add tabIndex={0} to this span?
I forgot to add it in my pr

**Review by d-rita — COMMENTED** [review]
Couple of thoughts (might be out of scope of the tickets): 

1. Tested the `split button` and i think we should be able to Tab it - just like the button component and activate it when i press `Enter`. Since there are 2 buttons, I can tab to any of the 2 to activate what i want. The `Esc` functionality after viewing the dropdown content works as expected. 

2. Tested the `Tooltip`. I can move focus to the one with a button trigger by tabbing to it, but the tooltip does not show as expected when the trigger receives focus. The 'Esc' functionality does work for the button trigger example.

**Comment by alaa-yahia** [comment]
> Couple of thoughts (might be out of scope of the tickets):
> 
> 1. Tested the `split button` and i think we should be able to Tab it - just like the button component and activate it when i press `Enter`. Since there are 2 buttons, I can tab to any of the 2 to activate what i want. The `Esc` functionality after viewing the dropdown content works as expected.
> 2. Tested the `Tooltip`. I can move focus to the one with a button trigger by tabbing to it, but the tooltip does not show as expected when the trigger receives focus. The 'Esc' functionality does work for the button trigger example.

@d-rita The issue with the Tooltip arises from the missing tabIndex. Since it is a <span> element, it will not receive focus by default. Please verify if it now works as expected after adding the tabIndex.
Thank you

**Comment by d-rita** [comment]
@alaa-yahia Tooltip works as expected 🎉

**Review by d-rita — APPROVED** [review]

**Comment by Chisomchima** [comment]
@d-rita your concerns have been added to a new (ticket)[https://dhis2.atlassian.net/browse/LIBS-611] and implementations can be found on this (PR)[https://github.com/dhis2/ui/pull/1512]

**Pull request merged**